tests/unit/multiSig.js

```diff
@@ -90,33 +90,39 @@ describe('Unit/MultiSig:', () => {
             await multiSigSecp.addPublicKeys(publicKeys)
             await expect(
                 multiSigSecp.signMulti(['0000ffeebabe', '0000ffeebabe'])
-            ).to.eventually.be.fulfilled;
+            ).to.eventually.have.length(2);
         });
     });
 
     describe('Aggregate and Verify Signatures', () => {
         it('Fail: Reject Aggregate Signatures when called with invalid argument', async () => {
             await expect(
                 multiSigSecp.aggregateSignatures(null)
-            ).to.eventually.be.rejectedWith(Error);
+            ).to.eventually.be.rejectedWith('BNAggregate.signatures: ');
         });
 
         it('Fail: Reject Aggregate Multi Signatures when called with invalid argument', async () => {
             await expect(
                 multiSigSecp.aggregateSignaturesMulti(null)
-            ).to.eventually.be.rejectedWith(Error);
+            ).to.eventually.be.rejectedWith('BNAggregate.signaturesMulti: ');
         });
 
         it('Fail: Reject Verify Aggregate when called with invalid argument', async () => {
             await expect(
                 multiSigSecp.verifyAggregate(null)
-            ).to.eventually.be.rejectedWith(Error);
+            ).to.eventually.be.rejectedWith('BNAggregate.verifyAggregateSingle: ');
```

Comment:
For the above three error checks, use the expected error.

In the current state these will always pass because the root error string is present, but they may be failing for reasons not expected. 

For example imagine this throws with " Error: BNAggregate.verifyAggregateSingle: some other reason"

In it's current state it would be assumed it's failing for the correct reason, however we want to provide an explicit test case that will let us know if an invalid object is the reason. If the underlying code were to change and remove the check for invalid arguments, we can catch it immediately this way.